interface/src/app/gene/searchMany/searchMany.js

```diff
@@ -25,6 +25,15 @@ angular.module('adage.gene.searchMany', [
               // be obtained from the ML model. This is the same as
               // the issue in geneSearchForm (also with $scope.organism).
               $scope.organism = 'Pseudomonas aeruginosa';
+
+              // '$scope.autocomplete.autocomlete' holds a boolean value,
+              // of whether or not the user wants to use autocomplete
+              // search to look up a few genes. If not, then the autocomplete
+              // search panel will disappear, and the panel to search many
+              // genes will appear. We need to send the value itself inside
+              // an object so that it gets sent properly back and forth
+              // through the corresponding $scopes.
+              $scope.autocomplete = {autocomplete: true};
```

Comment:
Attached is a tarball file of my test. It is based on an example in Chapter 11 of **AngularJS: Up & Running" book. If you click the buttons **toggle dhu** on index.html, it will toggle a boolean variable and show/hide a line **hello from dhu**. I tested the directive with both link function and controller.
[test.tar.gz](https://github.com/greenelab/adage-server/files/652890/test.tar.gz)